lib/src/types.ts

```diff
@@ -1,19 +1,40 @@
 import ow from 'ow';
-import KeyPair from './key_pair';
-
-export enum Network {
-    Mainnet = 'Mainnet',
-    Testnet = 'Testnet',
-    Devnet = 'Devnet',
-}
-
-export const CustomTypes = {
-    KeyPair: ow.object.validate((value) => ({
-        validator: value instanceof KeyPair,
-        message: `Expected value to be an instance of KeyPair, got ${value.constructor.name}`,
-    })),
-    Network: ow.string.validate((value) => ({
-        validator: value === 'Mainnet' || value === 'Testnet' || value === 'Devnet',
-        message: `Expected value to be one of the network variants (Mainnet, Testnet, Devnet)`,
-    })),
+import BigNumber from 'bignumber.js';
+
+import { Network } from './network';
+
+const native = require('../../native');
+
+export const owBigNumber = ow.object.validate((value: object) => ({
+    validator: BigNumber.isBigNumber(value),
```

Comment:
our Coin is smaller range compared to integer64,
maybe you can add more strict range check here
assert(value <= OutCoinMax)

Reply:
Good point, I will add extra condition

Reply:
Max coin condition added below